scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -121,9 +120,18 @@ RUN cd "$HOME" && \
 		make && make install && \
 		rm -rf etc/config.cache; \
 	done && \
-	cd "$HOME" && rm -rf newlib-xtensa && \
 	echo "Stage3: newlib-xtensa is done!"
 
+# log all hashes for each repo to a file
+ARG SOF_WORK="$HOME"/work
+ARG REPOS="$SOF_WORK/alsa/alsa-lib $SOF_WORK/alsa/alsa-utils $HOME/xtensa-overlay $HOME/crosstool-ng $HOME/newlib-xtensa"
```

Comment:
This will fail for anyone who has whitespace in HOME or SOF_WORK but considering how few people build this don't bother :-)

(a bash array would probably work)

Reply:
>  i can try with a bash array later

Don't bother. Add a FIXME if you feel really guilty :-)

Reply:
Added a FIXME